### DISCUSSION TITLE
How to setup delve?

### DISCUSSION THREAD

**Discussion by Jomik** [jay-babu/mason-nvim-dap.nvim#68] [Q&A] [answered]
What is the simplest way to setup `delve` ? I expected being able to just pass `delve` to `ensure_installed` and it would install and run the `automatic_setup` as well?
This does not seem to work for my case though. I actually do not even seem to get `delve` installed.
I use 
https://github.com/LazyVim/LazyVim/blob/5f6913c0dbac1026db70534e537c49d74f038ca1/lua/lazyvim/plugins/extras/dap/core.lua
and added this to my config.
```lua
  {
    "mfussenegger/nvim-dap",
    dependencies = {
      {
        "jay-babu/mason-nvim-dap.nvim",
        opts = {
          ensure_installed = {
            "delve",
          },
        },
      },
    },
  },
```

**Comment by jay-babu** [comment, marked as answer]
See Handlers section at the bottom: https://github.com/jay-babu/mason-nvim-dap.nvim
